src/index.ts

```diff
@@ -0,0 +1,3 @@
+export { Logger } from './logger';
```

Comment:
We probably want to export more stuff. E.g. `Event`, since we want to be able to use the types from `slf` when we implement drivers (or migrate them to TS). E.g. `slf-debug` will need some of the types.